### ISSUE TITLE
Dask Error: distributed.core - ERROR - addresses should be strings or tuples, got None

### ISSUE THREAD

**Issue by devin-petersohn** [modin-project/modin#1722]
The default to pandas test has been failing recently on Dask with Windows. I have searched and the errors thrown do not appear to be known issues at the moment.

Link to test failure: https://github.com/modin-project/modin/runs/862907959?check_suite_focus=true

**Comment by devin-petersohn** [comment]
```
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.7.8\x64\lib\site-packages\distributed\utils.py", line 656, in log_errors
    yield
  File "C:\hostedtoolcache\windows\Python\3.7.8\x64\lib\site-packages\distributed\scheduler.py", line 2208, in remove_worker
    address = self.coerce_address(address)
  File "C:\hostedtoolcache\windows\Python\3.7.8\x64\lib\site-packages\distributed\scheduler.py", line 4946, in coerce_address
    raise TypeError("addresses should be strings or tuples, got %r" % (addr,))
TypeError: addresses should be strings or tuples, got None
distributed.core - ERROR - addresses should be strings or tuples, got None
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.7.8\x64\lib\site-packages\distributed\core.py", line 513, in handle_comm
    result = await result

```

**Comment by devin-petersohn** [comment]
Issue was not resolved with #1723

**Comment by amyskov** [comment]
It was found out, that Modin CI fails with the latest versions of Dask and Distributed packages, so versions of these packages are downgraded to the previous versions for now.
